cmd/traffic/cmd/manager/internal/mutator/agent_injector.go

```diff
@@ -0,0 +1,234 @@
+package mutator
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	admission "k8s.io/api/admission/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
IMO, importing things as `v1` is an anti-pattern.  Say what it's v1 of! i.e. `corev1`